packages/main/src/SegmentedButton.hbs

```diff
@@ -0,0 +1,8 @@
+<div
+	@click="{{_onclick}}"
+	class="ui5-segmentedbutton-root"
+>
+	{{#each buttons}}
+		<slot name="{{this._individualSlot}}"></slot>

```

Comment:
I am wondering if we need the individual slot assignment as we do not wrap the buttons in spans or anything else?

```html
<div
	@click="{{_onclick}}"
	class="ui5-segmentedbutton-root"
>
    <slot></slot>
</div>
```

Reply:
Done.